doc/api/process.md

```diff
@@ -1178,6 +1178,9 @@ And `process.argv`:
 ['/usr/local/bin/node', 'script.js', '--version']
 ```
 
+In Worker threads, this property is not inherited, but explicitly passed
+at the construction time.
```

Comment:
It is inherited by default, though. It probably makes sense to link to the `Worker` constructor here for the detailed behavior, just like we do in other places in this file.

Reply:
@addaleax - when I ran this below code, I got an empty array. Doesn’t it mean that `process.execArgv` is not inherited to the worker?
```js
const { Worker } = require(‘worker_threads’);
const worker = new Worker(‘console.log(process.execArgv)’, {eval: true})
```

Reply:
```js
$ cat execargv.js 
const { Worker } = require('worker_threads');
const worker = new Worker('console.log("worker", process.execArgv)', {eval: true})
console.log("parent", process.execArgv)
$ node execargv.js 
parent []
worker []
$ node --title=foo execargv.js 
parent [ '--title=foo' ]
worker [ '--title=foo' ]
```

So yes, they are inherited. If you see an empty array in the Worker, and no `execArgv` option is passed to the `Worker` constructor, then you will also see an empty array in the parent thread :)